packages/lib/server/repository/profile.ts

```diff
@@ -36,6 +36,23 @@ const membershipSelect = Prisma.validator<Prisma.MembershipSelect>()({
 });
 
 const log = logger.getSubLogger({ prefix: ["repository/profile"] });
+const organizationSettingsSelect = Prisma.validator<Prisma.OrganizationSettingsSelect>()({
+  allowSEOIndexing: true,
+  orgProfileRedirectsToVerifiedDomain: true,
+});
+const organizationWithSettingsSelect = {
+  id: true,
+  slug: true,
+  name: true,
+  metadata: true,
+  logoUrl: true,
+  calVideoLogo: true,
+  bannerUrl: true,
+  isPlatform: true,
+  organizationSettings: {
+    select: organizationSettingsSelect,
+  },
+};
```

Comment:
NIT: Since it is the same
```suggestion
const organizationWithSettingsSelect = {
  ...organizationSelect,
  organizationSettings: {
    select: organizationSettingsSelect,
  },
};
```